package.json

```diff
@@ -86,7 +87,8 @@
     "vue-timers": "^2.0.4",
     "vuex": "^3.2.0",
     "vuex-class-modules": "^1.1.2",
-    "web3-utils": "1.0.0-beta.55"
+    "web3-utils": "1.0.0-beta.55",
+    "fingerprintjs2": "^1.8.6"
```

Comment:
please use for example `yarn add fingerprintjs2` command for add new dep, without manually declare it in `package.json` file

Reply:
what's the difference? yarn add just installs package and adds this line in package.json

Reply:
sort order of deps in package.json